airbyte-integrations/connectors/source-postgres/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/PostresSourceComprehensiveTest.java

```diff
@@ -346,6 +346,14 @@ protected void initTests() {
             .addExpectedValues("99999", "5.1", "0", null)
             .build());
 
+    addDataTypeTestData(
```

Comment:
thanks for adding the extra values for the test! We want this to be in **CdcPostresSourceComprehensiveTest.java** rather than this file though since that file runs the datatype checks with CDC enabled